.gitignore

```diff
@@ -2,4 +2,13 @@ leelaz
 *.o
 *.d
 training/tf/venv
-weights.txt
\ No newline at end of file
+weights.txt
+.idea
```

Comment:
Remove the .idea (we had a discussion that we're not going to put everyone's favorite IDE/editor in here) and duplicate the VS2015 parts to VS2017.

Reply:
OK. done.